core/src/main/resources/static/js/loginSaas.js

```diff
@@ -1,75 +0,0 @@
-'use strict'
```

Comment:
we need this file, as its used in oauthLogin.html 

which is referred in template controller
@RequestMapping(value = "/login", method = RequestMethod.GET)
  public String login(
      ModelMap model,
      HttpServletRequest request,
      HttpServletResponse response,
      AbstractAuthenticationToken abstractAuthenticationToken) {
    if (ssoEnabled.equals("true")) {
      return UriConstants.OAUTH_LOGIN;
    }

Reply:
I think it is only using `getBasicInfo` from this file, so probably we can keep it with only the `getBasicInfo` method and rename it `oauthLogin.js` ?

Reply:
agree, we can rename it